src/main/scala/com/github/gvolpe/http4s/StreamUtils.scala

```diff
@@ -1,14 +1,14 @@
 package com.github.gvolpe.http4s
 
 import cats.effect.Sync
-import fs2.Stream
+import fs2.{RaiseThrowable, Stream}
 
 trait StreamUtils[F[_]] {
```

Comment:
I'd get rid of this and probably just define methods at the effect level if needed. Eg. `env` and `putStrLn`.

Reply:
done